langchain4j-weaviate/src/main/java/dev/langchain4j/store/embedding/weaviate/WeaviateEmbeddingStore.java

```diff
@@ -81,6 +82,7 @@ public WeaviateEmbeddingStore(
             String objectClass,
             Boolean avoidDups,
             String consistencyLevel,
+            String metadataParentKey,
```

Comment:
```suggestion
            String metadataFieldName,
```

We already have a `textFieldName` parameter, perharps let's use `metadataFieldName` for consistency? Could you please also move it closer to the `textFieldName` in the list of params and javadoc?